src/Raster/RasterDevice.h

```diff
@@ -1,21 +1,25 @@
-#pragma once
+
+#ifndef RASTER_DEVICE_H
+
+#define RASTER_DEVICE_H
+
 
 #include <bld_system.h>
 #include <BUIxc.h>
+#include <BBLibc/Name.h>
+#include <BBLibc/Color.h>
+#include <BBLibc/ParticleGType.h>
+#include <BBLibc/BitMap24.h>
 #ifdef RASTER_LIB_EXPORT
 #define BUILD_LIB
 #endif
 #include <export.h>
 #undef BUILD_LIB
 
 class B_ViewPose;
-class B_BitMap24;
 class B_CameraView;
-class B_Name;
-class B_Color;
 class B_Font;
-struct B_Particle;
-class B_ParticleGType;
```

Comment:
Still unresolved.

Reply:
That's strange. Because it has it this commit: #[ed41cfa](https://github.com/smartblade/BldMystery/pull/5/commits/ed41cfa8d18e357a02748a2bcdb0c00ebf9277be)

Reply:
Also `Name`, `Color` and `Bitmap24`.

Reply:
Understood.

Reply:
`B_Color` was needed, without it, it does not compile. So I left it.